ingress-nginx/etc/nginx/lua/balancer.lua

```diff
@@ -0,0 +1,374 @@
+local ngx_balancer = require("ngx.balancer")
```

Comment:
@strongjz the *.lua files that are being added here, I wonder if they are coming from an existing git repo that we could clone in the melange pipeline rather than duplicating here?  Just thinking it might make it hard to maintain these if / when they change?

Reply:
I was going to rethink how this was done with @imjasonh, havent had that conversation yet. It does come the ingress-nginx repo, here https://github.com/kubernetes/ingress-nginx/tree/main/images/nginx/rootfs